### DISCUSSION TITLE
Hello! I would like to be a collaborator.

### DISCUSSION THREAD

**Discussion by sardorml** [ParanoidUser/yolo#1284] [Subscribe]
Hello! I would like request to be a collaborator.

**Comment by ParanoidUser** [comment]
Ahoy, @sardorml! 👋

Welcome aboard our project! 🚢 I'm sending the collaborator invitation your way right now. Feel free to share your thoughts on any cool ideas you have. Any questions? Just let me know. Let's get this show on the road!

Fair winds

**Comment by lord-dubious** [comment]
how does this work
